Source/Resources/Content/AadIam/GroupSettings/Configuration.json

```diff
@@ -1,5 +1,6 @@
 {
   "allUsersGroupEnabled": false,
+  "description":"What is it: Configures group membership management options e.g., the ability for group owners to approve requests to join groups, restrictions for users to see group membership, and the ability for users to create security groups. \n Simeon Notes: Simeon's baseline does not allow group owners to approve requests and does not allow security group creation. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/groups-self-service-management",
```

Comment:
**What does this do?**
Configures group membership management options. The baseline does not allow group owners to approve requests, does not allow security group creation, and restricts users from seeing group membership.

**Why should you use this?**
If you want to have a more secure group settings environment.

**What is the end user impact?**
Owners cannot approve requests, security groups cannot be created, and users cannot see group membership.

**Learn more**
[Users, groups, and roles](https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/groups-self-service-management)